src/external_data_ifaces_impl.cpp

```diff
@@ -109,6 +112,42 @@ sdbusplus::async::task<> ExternalDataIFacesImpl::fetchBMCPosition()
     co_return;
 }
 
+sdbusplus::async::task<>
+    ExternalDataIFacesImpl::createErrorLog(const std::string& errMsg,
+                                           const ErrorLevel& errSeverity,
+                                           const json& calloutsDetails)
+{
+    try
+    {
+        error_log::FFDCFileInfoSet ffdcFileInfoSet;
+        if (!calloutsDetails.is_null())
+        {
+            error_log::FFDCFile file(error_log::FFDCFormat::JSON, 0xCA, 0x01,
```

Comment:
I think its not required but we could revisit to remove it if its not required since the callout is added as part of message registry now.